gpkit/tests/t_nomials.py

```diff
@@ -1,7 +1,7 @@
 import math
 import unittest
 from gpkit import Monomial, Posynomial, Signomial
-import gpkit
+from gpkit import enable_signomials, disable_signomials
```

Comment:
Is it straightforward to explain why? (Unit tests all pass with the changes above.)